### PR TITLE
Load transfer fork

### PR DESCRIPTION
**Pull request by ehariton**
solves issue #82

### CODE REVIEW

**Comment by johnjasa** [comment]
These failed on some of the deriv and optimization tests. They're pretty close to the tolerance, though. Could you check these cases locally and see if you can spot something?

**Comment by ehariton** [comment]
resolved some issues with loadsB and loadsA not getting set to zero during def compute()

**Comment by johnjasa** [comment]
Pulled in by #206

File: openaerostruct/transfer/load_transfer.py [hunk 1]
```diff
@@ -54,76 +59,288 @@ def setup(self):
             self.fem_origin = (x_upper[0]  * (y_upper[0]  - y_lower[0]) +
                                x_upper[-1] * (y_upper[-1] - y_lower[-1])) / \
                              ((y_upper[0]  -  y_lower[0]) + (y_upper[-1] - y_lower[-1]))
+        self.w1 = w1 = 0.25
+        self.w2 = w2 = self.fem_origin
 
-        self.add_input('def_mesh', val=np.ones((self.nx, self.ny, 3)), units='m')
-        self.add_input('sec_forces', val=np.ones((self.nx-1, self.ny-1, 3)), units='N')
+        self.add_input('def_mesh', val=np.random.random((nx, ny, 3)), units='m')
+        self.add_input('sec_forces', val=np.random.random((nx-1, ny-1, 3)), units='N')
```
[the remark this answers is not in the document]
Comment (ehariton) [reply openaerostruct/transfer/load_transfer.py]: We did that initially so we could test the partial derivatives effectively. Now that those are complete, we could switch it back to ones or zeros. Ideally, all these values should be set in the test conditions anyways so they wouldn't revert to their defaults.

**Pull request closed without merging**